### ISSUE TITLE
Hide the "version upgrade" notice when you're already updating

### ISSUE THREAD

**Issue by ZackeryRSmith** [tristanisham/zvm#91]
Currently when running `zvm update` it will update as you'd expect, but still throw in the notice that you're on an outdated version. I think this is a bit silly and would likely be best to be removed.

```
$ zvm upgrade
Upgrading ZVM... 100% |█████████████████████████████████████████████████████████████████████████████████████████████| (8.6/8.6 MB, 23 MB/s)


  Notice

ZVM v0.7.3 is available. You are currently on v0.7.2.

Run zvm upgrade or download the latest release at
https://github.com/tristanisham/zvm/releases/latest
```

**Comment by tristanisham** [comment]
Honestly, true. We're working on redoing how ZVM is installed #92, so this might be something we can adjust @isFakeAccount.

**Comment by tristanisham** [comment]
Just fixed this with ZVM v0.7.4 :)
